package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "shop-analytics",
-  "version": "1.8.2",
+  "version": "1.8.3",
```

Please see https://semver.org about the naming convention. When you add a setting, this would be a minor version. 
```suggestion
  "version": "1.9.0",
```